packages/@okta/vuepress-site/guides/sign-users-out/redirect-after-sign-out/index.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
This section is applicable only when you sign out from Okta. I'm still not 100% convinced if this should be a separate section or part of the `Okta Sign Out` section. @nbarbettini thoughts?

Reply:
Makes sense as part of the _Sign out of Okta_ step.

Reply:
Thinking about it some more, we don't yet have a consistent story across all libraries for logout. (@swiftone's tech spec is what we need!)
Let's delete this step for now and each language can put the instructions in the _Sign out of Okta_ step.